qiskit_ibm_provider/ibm_backend.py

```diff
@@ -883,6 +883,16 @@ def session(self) -> Session:
 
     def cancel_session(self) -> None:
         """Cancel session. All pending jobs will be cancelled."""
+        if self._session:
+            self._session.cancel()
+            if self._session.session_id:
+                self.provider._runtime_client.cancel_session(self._session.session_id)
```

Comment:
`self.provider._runtime_client.cancel_session` doesn't exist in this repo so it'll have to be added there as well

Reply:
You are right, I shouldn't have submitted for review so quickly. I copied the implementation from `runtime`. Also added a test, though it is pretty trivial. We will unite this functionality back when uniting the two repos.